### DISCUSSION TITLE
Named Imports

### DISCUSSION THREAD

**Discussion by PhantomKnight287** [vercel/next.js#32397] [Ideas]
Why Nextjs does not support named imports like
```js
import {useRouter,Head,Image} from 'next';
```
as default imports just add more import statements in the code.

**Comment by jrmarqueshd** [comment]
I think that, this is not possible because next not export all libraries by default.

**Reply by PhantomKnight287** [reply]
No I mean next almost export all things as default. Can't they make them named imports as this will reduce the number of import statements we've to  write.
